canonical/ubuntu.com
# Build /ai/consulting and remove /ai/services

## Done

- Built page as per [doc](https://docs.google.com/document/d/1Zj9agFHVT5pEXapazCcoh_o2v0jEU1Oe7BmdvDbDGrg/edit) and [design](https://www.figma.com/file/sGeCjFvA1kV9Kmvg2S2VBn/23.10-U.com---AI-bubble?node-id=248%3A1446&mode=dev)
- Removed /ai/services as per instructions in doc
- Added redirect: /ai/services -> /ai/consulting
- **Note: The class for this suru hasn't been merged into the feature branch yet, hence why you don't see it here. When https://github.com/canonical/ubuntu.com/pull/13146 is merged the suru will show up as expected**

## QA

- View the site locally in your web browser at: https://ubuntu-com-13154.demos.haus/ai/consulting
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check against doc and design

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-5662
